library/core/Cargo.toml

```
@@ -43,6 +43,7 @@ check-cfg = [
     'cfg(bootstrap)',
     'cfg(no_fp_fmt_parse)',
     'cfg(stdarch_intel_sde)',
+    'cfg(target_os, values("rtems"))',
```

could you also add this comment, so that the release team can remove it when doing the bootstrap bump
```suggestion
    # #[cfg(bootstrap)] rtems
    'cfg(target_os, values("rtems"))',
```

---

Thank you for the suggestions. I am not familiar with these constructs, but they were necessary since the latest changes in master, so I cargo-culted a bit.